### PR TITLE
list only displays i have access to

### PR DESCRIPTION
**Pull request by settinghead**

### CODE REVIEW

**Comment by settinghead** [comment]
@tejohnso this PR is a bit rushed but it should be working for the most part (the only thing missing is removing a display, which is my next card and only half-implemented). 

Thought I should get what I have merged first in case I won't block anyone.

Please review. @Rise-Vision/delivery FYI. Thanks.

File: src/components/displays/display_list.service.js [hunk 1]
```diff
@@ -1,20 +1,76 @@
...
+    async function getMyDisplays() {
+
+      if ( await authService.amIRiseAdmin() ) {
+        return await getAllDisplays();
+      } else {
+
+        const myUserId = await authService.getMyUserId(),
+          displayArr = new DisplaysForUser(
+          root.child( `users/${myUserId}/displays` ), myUserId
+        )
```
Comment (tejohnso) [inline src/components/displays/display_list.service.js]: What are the advantages of this extended service over a simple $firebaseObject(users/userid/displays) call?

Comment (settinghead) [inline src/components/displays/display_list.service.js]: @tejohnso This is for joining two parts of data for real time pub/sub (in this case `users/userId/displays/displayId` and `displays/displayId`). This is the only way I could find to correctly update two joining paths for all subscribing clients (i.e. when a path is added/updated, also reload its associating parts).
The reason for `$firebaseArray` over `$firebaseObject`: array has orderi information, which we will need down the road (though in most browsers like FF and Chrome the order of object keys are preserved based on the order they are assigned, however this is not a 100% guarantee).

File: src/components/auth/auth.js [hunk 2]
```diff
@@ -9,10 +9,21 @@ const PROTECTED_PATHS = [ "profile", "lists" ],
   ] )
 .component( "authFormGoogle", authFormGoogleComponent )
 .service( "authService", AuthService )
+.constant( "emailToUserKey", emailToUserKey )
+.constant( "userKeyToEmail", userKeyToEmail )
```
Comment (settinghead) [inline src/components/auth/auth.js]: @tejohnso good idea.

File: src/components/displays/display_list.service.js (hunk 1 above)
Comment (tejohnso) [inline src/components/displays/display_list.service.js]: @settinghead I'm not sure how they'd get out of sync.   When would we be updating `users/userId/displays/displayId` without also updating `displays/displayId` ?

Comment (settinghead) [inline src/components/displays/display_list.service.js]: @tejohnso I see your point. I probably overcomplicated it. Will see if the code can be simplified.

**Pull request merged**